core/store/migrate/migrations/0115_migrate_upkeep_id_to_big_int.sql

```diff
@@ -0,0 +1,48 @@
+-- +goose Up
```

Comment:
will also need to update the job spec for pretyID

Reply:
prettyID is derived directly from upkeepID, so it doesn't need a column in the DB

Reply:
I meant updating the observation source in job spec.

to change ```txMeta="{\\"jobID\\":$(jobSpec.jobID),\\"upkeepID\\":$(jobSpec.upkeepID)}"```
to ```txMeta="{\\"jobID\\":$(jobSpec.jobID),\\"upkeepID\\":$(jobSpec.prettyID)}"``` within DB

Reply:
yes! good catch.